package.json

```diff
@@ -40,6 +40,7 @@
     "paratii-protocol": "github:Paratii-Video/paratii-protocol#45aadcf300f1ad184d6d3db8f9c4b3d438b6f658",
     "promisify-node": "^0.4.0",
     "pull-filereader": "^1.0.1",
+    "request": "^2.87.0",
```

Comment:
we are using "fetch" in other places, so if we can, we should use that.

Reply:
Which one of these?

https://www.npmjs.com/package/whatwg-fetch
https://www.npmjs.com/package/fetch
https://www.npmjs.com/package/node-fetch
https://www.npmjs.com/package/isomorphic-fetch

Reply:
check in `package.json` :-)  (it's `isomorphic-fetch`)

Reply:
Done